Documentation/01 Getting Started.md

```diff
@@ -181,17 +191,17 @@ $ open diff.html
 
 Well that still looks like a great optimization to me -- `Array` is slightly faster at small sizes, but as the data grows, it cannot keep up with `Deque` at all. (By the way, I'm pretty sure that initial 10% difference will be reduced as the Collections package improves! `Array` has already received more than half a decade's worth of performance work, but work on `Deque` is just getting started. 😉)
 
-The `results compare` command gets even more powerful once we have more than one benchmark. It is invaluable for getting us an objective (dare I say, ruthless) measure of how a potential change affects performance for our collection implementations. 
+The `results compare` command gets even more powerful once we have more than one benchmark. It is invaluable for getting us an objective (dare I say, ruthless) measure of how a potential change affects performance for our collection implementations.
 
 ### Benchmark Libraries
 
-If you're anything like me, once you get the hang of writing benchmarks, it's difficult to stop. For example, the `Collections` package has several hundred (and counting!) benchmark definitions, each capturing a different aspect of data structure behavior. 
+If you're anything like me, once you get the hang of writing benchmarks, it's difficult to stop. For example, the `Collections` package has several hundred (and counting!) benchmark definitions, each capturing a different aspect of data structure behavior.
 
 With so many benchmarks, we need a way to organize them into a series of thematic charts that make sense -- trying to render them all on a single chart makes for interesting glitch art, but it isn't very practical:
 
 ![chaos.png](Assets/chaos.png)
 
-Here is where benchmark libraries come in! They allow us to maintain a collection of interesting charts, organized into hierarchical sections, with nice descriptive titles. Once we have a library, we can easily re-run the specific benchmarks it contains, and render the result in a nice document.
+Here is where benchmark libraries come in! They allow us to maintain a collection of interesting charts, organized into hierarchical sections, with nice descriptive titles. Once we have a library, we can re-run the specific benchmarks it contains, and render the result in a nice document.
```

Comment:
Hm, the whole point here is that rerunning the benchmarks is easier than it would be if we were doing it by hand. Losing the adjective would undersell the feature.

```suggestion
Here is where benchmark libraries come in! They allow us to maintain a collection of interesting charts, organized into hierarchical sections, with nice descriptive titles. Once we have defined a library, we can easily re-run the specific benchmarks it contains, and render the result in a nice document. We don't need to keep track of what combinations of tasks make the most useful charts, or to remember to run every task we're going to need -- the library takes care of administering these details.
```